man/ocitools-generate.1.md

```diff
@@ -79,6 +79,18 @@ inside of the container.
 **--mount-cgroups**=[rw|ro|no]
   Mount cgroups.  The default is `no`.
 
+**--mount-label**=[=*MOUNTLABEL*]]
```

Comment:
Redundant equal signs?  I'd guess this should be **--mount-label**=[*MOUNTLABEL*].